validator/src/validator.rs

```diff
@@ -743,7 +793,10 @@ where
                 Ok(ConsensusEvent::Lost)
                 | Ok(ConsensusEvent::Established {
                     synced_validity_window: false,
-                }) => self.pause(),
+                }) => {
+                    self.pause();
+                    self.recent_block_timestamps.clear();
```

Comment:
I *think* this shouldn't be necessary, as the `self.recent_block_timestamps` should be updated correctly even in case of lost consensus.